src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx

```diff
@@ -116,7 +116,7 @@ The New Relic Kubernetes integration has the following requirements:
 * A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
 * Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
 * [`kube-state-metrics` version 1.9.8](https://github.com/kubernetes/kube-state-metrics) running on the cluster.
-* When using CRI-O as the container runtime, the [processes](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
+* When using CRI-O as the container runtime, the [processes](/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
```

Comment:
I know some of these links weren't part of your sweep, but should they be made relative too?

Reply:
I'm inclined to not block on making the other links relative, there are around ~2000 absolute links on the site that probably "should" be relative links and I feel like it's a pretty bottomless rabbithole.